core/dbt/task/run.py

```diff
@@ -717,30 +715,20 @@ def handle_microbatch_model(
 
         batch_results: List[RunResult] = []
         batch_idx = 0
-
-        # Run first batch runs in serial
-        relation_exists = self._submit_batch(
-            node, relation_exists, batches, batch_idx, batch_results, pool, parallel=False
-        )
-        batch_idx += 1
```

Comment:
I think it's actually preferable to the first batch separated out. It makes it mirror how the last batch is separated out. We could also have an optional arg `force_sequential` (defaulted to `False`) which would skip the `should_run_in_parallel` check in `_submit_batch`